src/CachePlugin.php

```diff
@@ -441,4 +452,22 @@ private function getETag(CacheItemInterface $cacheItem)
             }
         }
     }
+
+    /**
+     * Call the reponse mutator, if one is set.
+     *
+     * @param RequestInterface  $request
+     * @param ResponseInterface $response
+     * @param bool              $cacheHit
+     *
+     * @return ResponseInterface
+     */
+    private function handleResponseMutator(RequestInterface $request, ResponseInterface $response, $cacheHit)
+    {
+        if (null !== $this->config['response_mutator']) {
+            $response = $this->config['response_mutator']->mutate($request, $response, $cacheHit);
```

Comment:
i think this should be a list of mutators.

Reply:
Good suggestion, will update.

Reply:
Though, not totally familliar with the resolver... is there any better way of stating that the array must be of generic type `CacheListener`? Right now I just have `$resolver->setAllowedTypes('cache_listeners', ['array']);`.